### PR TITLE
Updates to Home.Md

### PR DESCRIPTION
**Pull request by reme-dy**
Rework of `Home.md`:

* Moved the CLI arguments to a new file, `Execution.md`
* Updated sidebar with link to Execution
* Added FAQ elements from README.md
* Added a couple more FAQs
* Sprinkled Discord link
* JSON online editor link
* README.md is a clone of Home.md

### CODE REVIEW

File: Home.md [hunk 1]
```diff
@@ -1,55 +1,69 @@
...
+* [Slack](https://github.com/kvangent/PokeAlarm/wiki/Slack)
+* [Telegram](https://github.com/kvangent/PokeAlarm/wiki/Telegram) 
+* [Twilio (SMS)](https://github.com/kvangent/PokeAlarm/wiki/Twilio-(SMS))
+* [Twitter](https://github.com/kvangent/PokeAlarm/wiki/Twitter)
 
-Pushbullet support will likely be discontinued after August 1st because of the change of services to premium. 
+## Support
 
-If you are experience issues with the alarm or would like to see new features, please open a ticket here on github. 
+### [Discord Link](https://discordapp.com/invite/am66rag) <---- Visit us here first!!!
```
Comment (kvangent) [inline Home.md]: Let's shorten this to 'Discord' - link feels redundant

File: Home.md [hunk 2]
```diff
@@ -1,55 +1,69 @@
...
+For questions about set up, please visit us at our [Discord channel](https://discordapp.com/invite/am66rag). Stick to the #general and #troubleshooting chats and avoid sending private messages to devs. We're working hard, we promise!
```
Comment (kvangent) [inline Home.md]: set up - should be one word
Add a 'Please' infront of Stick to the #general

File: Home.md [hunk 3]
```diff
@@ -1,55 +1,69 @@
...
 
-If you are experiencing trouble with setup or other user difficulties, please see the reddit thread.
+If you are experiencing issues with the alarm or would like to see new features, please open a ticket on github [here](https://github.com/kvangent/PokeAlarm/issues/new). Be sure to complete the included suppport template and provide as much information as possible.  **Support tickets that do not fully complete the request template may be closed without notice.**
 
-##Command Line Arguments
+Our PokeAlarm reddit thread is located [here](https://www.reddit.com/r/pokemongodev/comments/4wl5my/pokealarm_v21_external_alerts_from_your_favorite).
```
Comment (kvangent) [inline Home.md]: Let's just forget references to the reddit thread for now. Discord is easier to deal with

File: Home.md [hunk 4]
```diff
@@ -1,55 +1,69 @@
...
+#### I am receiving an error about JSON input from PokeAlarm (e.g., "expecting delimiter"). What gives?
 
-## Common Problems
+* If you are a Windows user, stop using notepad and start using Notepad++. Make a fresh copy of the alarms.json and remake your changes. If you aren't a Windows user (or you are already using Notepad++) check your JSON format with a JSON formatter for issues ([jsoneditoronline.org](http://www.jsoneditoronline.org) is an excellent site to start.)
 
-#### Unable to decode JSON object
+#### Can I run multiple simultaneous alarms services?
 
-Make sure you aren't using notepad to edit the files (Notepad++ is a good alternative). Also, check to make sure your json formatting is correct. You can check your format [here](http://www.jsoneditoronline.org/).
+* Yes. You may configure as few or as many simultaneous alarm services in `alarms.json` like Twitter, Slack and Telegram.  For example, you can post to 2 Twitter accounts, 3 Slack channels and have a Twilio SMS service running from one `alarms.json` configuration. Visit the Alarm Configuation wiki for details.
```
Comment (kvangent) [inline Home.md]: for more* details

File: Home.md [hunk 5]
```diff
@@ -1,55 +1,69 @@
...
 
-#### Problems with receiving in UTC time or Notifications not sending because 'time_left has passed'.
+Alternatively, you can run PokemonGo-Map with multiple webhooks and have multiple instances of PokeAlarm assigned to a different http://host:port (e.g., http://127.0.0.1:4000, http://127.0.0.1:4001, http://127.0.0.1:4002).
```
Comment (kvangent) [inline Home.md]: make it <host>:<port> for more clarity

File: Home.md [hunk 6]
```diff
@@ -1,55 +1,69 @@
...
 
-This error should now be corrected. Please make sure you have updated to the latest version of PokemonGo-Map. If you are still experiencing issues, please open a ticket. 
+#### I'm having issues with setting a location, it is not showing distance, maps are not showing up, walking directions, etc.
 
-#### Windows: socket.error: [Errno 10053] An established connection was aborted by the software in your host machine OR Linux: socket.error: [Errno 32] Broken pipe
+* These errors are all map related.  Sign up for a Google Maps API key and enable Static Maps in the console. See the Google Maps documentation [here](https://developers.google.com/maps/documentation/static-maps/intro). To test if your key is working, visit the URL below and replace `YOUR_API_KEY` with your Google Maps API key.
```
Comment (kvangent) [inline Home.md]: Need to add language to clarify - Geocoding API is only required for setting location and reverse location, Distance matrix API required for getting walking/bike/drive distance, and Static Map API for slack maps

File: Home.md [hunk 7]
```diff
@@ -1,55 +1,69 @@
...
+
+```javascript
+https://maps.googleapis.com/maps/api/staticmap?center=Brooklyn+Bridge,New+York,NY&zoom=13&size=600x300&maptype=roadmap&markers=color:blue%7Clabel:S%7C40.702147,-74.015794&markers=color:green%7Clabel:G%7C40.711614,-74.012318&markers=color:red%7Clabel:C%7C40.718217,-73.998284&key=YOUR_API_KEY
+```
+If your key is valid, you should see this same static map image in your browser:
+![](images/staticmaptest.png)
+
+If your key has issues, you will get the error `The Google Maps API server rejected your request. The provided API key is invalid.`
```
Comment (kvangent) [inline Home.md]: Might also get an error for API disabled - might be better just to reduce this section down to "Make sure you have an API Key with all the necessary API's enabled. For more information see Google Maps API Key" with link to another page on setup

File: Home.md [hunk 8]
```diff
@@ -1,55 +1,69 @@
...
 
-This error should now be corrected. Please make sure you have updated to the latest version of PokemonGo-Map. If you are still experiencing issues, please open a ticket.
 
 
+#### Problems with receiving in UTC time or Notifications not sending because 'time_left has passed'.
```
Comment (kvangent) [inline Home.md]: You can get rid of this

File: Home.md [hunk 9]
```diff
@@ -1,55 +1,69 @@
...
+
+* This error should now be corrected. Please make sure you have updated to the latest version of PokemonGo-Map. If you are still experiencing issues, please open a ticket. 
+
+#### Windows: socket.error: [Errno 10053] An established connection was aborted by the software in your host machine OR Linux: socket.error: [Errno 32] Broken pipe
```
Comment (kvangent) [inline Home.md]: Can get rid of this too

File: README.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+#### Can I run multiple simultaneous alarms services?
+
+* Yes. You may configure as few or as many simultaneous alarm services in `alarms.json` like Twitter, Slack and Telegram.  For example, you can post to 2 Twitter accounts, 3 Slack channels and have a Twilio SMS service running from one `alarms.json` configuration. Visit the Alarm Configuation wiki for details.
+
+Alternatively, you can run PokemonGo-Map with multiple webhooks and have multiple instances of PokeAlarm assigned to a different http://host:port (e.g., http://127.0.0.1:4000, http://127.0.0.1:4001, http://127.0.0.1:4002).
+
+#### I'm having issues with setting a location, it is not showing distance, maps are not showing up, walking directions, etc.
+
+* These errors are all map related.  Sign up for a Google Maps API key and enable Static Maps in the console. See the Google Maps documentation [here](https://developers.google.com/maps/documentation/static-maps/intro). To test if your key is working, visit the URL below and replace `YOUR_API_KEY` with your Google Maps API key.
```
Comment (kvangent) [inline README.md]: Lets reduce this to "Make sure you have an API-Key enabeled. For more information see the Google-Maps API Key page."

**Comment by reme-dy** [comment]
Yep it's in https://github.com/kvangent/PokeAlarmWiki/pull/2/commits/8f7e7a6955fd8a71ecc1354496e4904a1ed0b12a. "These errors are Google Maps API related. Ensure you have an API key with all the necssary APIs enabled.  Visit the Google Maps API Key wiki for more details."

File: Execution.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+## Alarm Configuration file (alarms.json)
+The `alarms.json` file is where you would customize your notifications.  It:
+
+* Enables/disables alarm services
+* Formats specific alarm notifications
+* Enables pokemon, gym, or lured pokestop to be notified
+
+Note that the default name of this alarm configuration file is `alarms.json`.  However, PokeAlarm allows you to select specific `*.json` configuration files with the `-c` or `--config` flag.
+
+See the Alarm Configuration wiki for setup and more details.
```
Comment (kvangent) [inline Execution.md]: Lets get a link to the wiki page here

File: README.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
+PokeAlarm is a third party extension of [PokemonGo-Map](https://github.com/PokemonGoMap/PokemonGo-Map) that allows you to receive external notifications via one or more message services.  Customized notifications are available by configuring the `alarms.json` file.
+
+We currently support the following services:
+* [Pushbullet](https://github.com/kvangent/PokeAlarm/wiki/Pushbullet) (support likely be discontinued soon due to changes in the API pricing structure)
```
Comment (kvangent) [inline README.md]: Pushover is missing
